charts/cp-schema-registry/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## cp-schema-registry-v7.3.1-bbc-2
+
+* Schema registry bugfix:
+    * chore: Upgraded Caddy server to 2.6.4 in the Schema Registry UI
+    * ref: Standardized configuration with a hard-coded config in the Docker image, no
+      entrypoint
+    * fix: `Oidc_*` Headers are not proxied anymore to the backend Schema
+      Registry. This reduces a lot the total header length, avoiding hitting
+      the max header length limit on the backend side.
```

Comment:
This chart is only for the schema-registry, so you can just list changes "directly", without a subsection:
```suggestion
* chore: Upgraded Caddy server to 2.6.4 for the UI
* ref: Standardized configuration with a hard-coded config in the Docker image, no
   entrypoint
* fix: `Oidc_*` Headers are not proxied anymore to the backend Schema
   Registry. This reduces a lot the total header length, avoiding hitting
   the max header length limit on the backend side.
```

Reply:
BTW: what do you call "ref" ?

Reply:
Ha :) :) :)
Thanks !